### DISCUSSION TITLE
Is There any possible ways to import the repositories which I had created to my frontend

### DISCUSSION THREAD

**Discussion by Siva20021** [community/community#39652] [Repositories]
### Select Topic Area

Question

### Body

Few applications(like Vercel ,HacktoberFest )  have this feature, whenever I Login/Signup through GitHub they can display the repositories which I had worked. So is there any possibility for me as well to access through those API. Is it a custom API or is it any API which anyone can have access through. If anyone can have access through can you please say me the steps to access it .

**Comment by airtower-luna** [comment]
The [repositories API](https://docs.github.com/en/rest/repos/repos) is probably a good starting point to look. If you need more help, please be more specific about what you're trying to do. :slightly_smiling_face:

**Reply by Siva20021** [reply]
Actually Since people do projects and some are lazy towards adding it to their Portfolio website so some organization which uses git to authenticate fetches our git commits so similar to that if we had a same api for fetching our repositories it will be useful for people to display it on their portfolio.

**Comment by VladCuciureanu** [comment]
https://api.github.com/users/USERNAME/repos would be a good place to start.

[Example](https://api.github.com/users/Siva20021/repos)

All that's left is for you to refine the JSON output in any way you like

**Reply by Siva20021** [reply]
Thanks a lot . I didn't knew this

**Reply by VladCuciureanu** [reply]
If this solves your question please mark it as answered
